src/sql-parser/tests/testdata/ddl

```diff
@@ -1734,63 +1734,71 @@ ALTER CLUSTER cluster SET (SIZE '1')
 ----
 ALTER CLUSTER cluster SET (SIZE = '1')
 =>
-AlterCluster(AlterClusterStatement { if_exists: false, name: Ident("cluster"), action: SetOptions([ClusterOption { name: Size, value: Some(Value(String("1"))) }]) })
+AlterCluster(AlterClusterStatement { if_exists: false, name: Ident("cluster"), action: SetOptions { options: [ClusterOption { name: Size, value: Some(Value(String("1"))) }], strategy: [] } })
+
+
+parse-statement
+ALTER CLUSTER cluster SET (SIZE '1') WITH ( WAIT FOR '1s' )
+----
+ALTER CLUSTER cluster SET (SIZE = '1') WITH (WAIT = FOR'1s')
```

Comment:
Is there any previous discussion on the option syntax?

Reply:
lots https://github.com/MaterializeInc/materialize/pull/26785

but I'm happy to revisit if you have some thoughts.

Reply:
No, I think it's fine as is.